### PR TITLE
Added time of day to window which makes it easier to pass time of day…

### PR DESCRIPTION
**Pull request by clare**
Added my time of day parser as I will use it for the config files for thermal-recorder

### CODE REVIEW

File: timeofday.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
```
Comment (mjs) [inline timeofday.go]: Wrong license and still says audiobait :)

File: timeofday.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+type TimeOfDay struct {
+	time.Time
+}
+
+const timeLayout = `15:04`
+const timeLayoutJson = `"` + timeLayout + `"`
+
+func (timeOfDay *TimeOfDay) UnmarshalJSON(bValue []byte) error {
```
Comment (mjs) [inline timeofday.go:19]: Do we actually use these with JSON anywhere?

File: timeofday_test.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
```
Comment (mjs) [inline timeofday_test.go]: same problem

File: timeofday.go (hunk 2 above)
Comment (clare) [reply timeofday.go:19]: Yes Json version is used in audiobait where I pulled this code from.

**Review by CameronRP — APPROVED** [review]

**Pull request merged**